### ISSUE TITLE
Spectral Panel - Intralipid Phantom MuA inaccurate

### ISSUE THREAD

**Issue by ccampb19** [VirtualPhotonics/Vts.Gui.Wpf#22]
Since water makes up the majority of the intralipid absorption spectrum, I would expect to see a water peak at 970nm. Also, I believe the peak at 700nm is spurious. At least, I don't see this peak in my nigrosin spectra or the lipid spectrum we use, or in my Intralipid measurements. Perhaps it's related to the water problem.

![image](https://user-images.githubusercontent.com/43229278/60526712-d99a5400-9cbe-11e9-8443-e9df22c35671.png)

**Comment by dcuccia** [comment]
Interesting, thanks Chris. What does it look like if you back down nigrosin to near zero, e.g. 0.0001?

**Comment by ccampb19** [comment]
Hey David. It's the same curve, scaled.
![image](https://user-images.githubusercontent.com/43229278/60528334-3ba88880-9cc2-11e9-9914-5189f65e0277.png)

**Comment by dcuccia** [comment]
Thanks. Looks like water is not in there at all. With an IL vol fraction of 0.01, the water fraction should be 0.99, resulting in an absorption of about 0.05/mm at roughly 979nm.

**Comment by ccampb19** [comment]
Right, and the water spectrum works in a custom tissue -- but then there's no nigrosin available to add.
